string-manipulation-1/src/get-words.js

```diff
@@ -0,0 +1,15 @@
+/* exported getWords */
+
+// compare if the length of string is less or equal to 0 to check for empty strings
+// which should return an empty array
+// if the length is larger than 0
+// divide the string at every space and return output
+
+function getWords(string) {
+  for (var i = 0; i <= string.length; i++) {
+    if (string.length <= 0) {
+      return [];
+    }
```

Comment:
This conditional isn't making use of `i` at all, you don't actually need the loop 🤔